query2.php

```diff
@@ -42,7 +42,7 @@
 		<div class="grid-container">

```

Comment:
Updated same images and graphics from query1.php on to query2.php, however replaced these with images that suit query2 context.